test/unit/store/map/index/actions.spec.js

```diff
@@ -187,29 +249,29 @@ describe('loadPointDataForLocation', () => {
     }
     getFromApi.mockResolvedValue(apiResult)
     await actions.loadPointDataForLocation(
-      { commit, state: {} },
-      { datasetIds: ['ab', 'cd'], locationId: 'ef' },
+      { commit, state },
+      { datasetIds: ['ab', 'cd'], locationId: 'loc1' },
     )
     expect(getFromApi).toHaveBeenCalledWith('timeseries', {
       datasetId: 'cd',
       endTime: moment()
         .add(5, 'days')
         .format('YYYY-MM-DDTHH:mm:ssZ'),
-      locationId: 'ef',
+      locationId: 'loc1',
       startTime: moment()
         .subtract(3, 'days')
         .format('YYYY-MM-DDTHH:mm:ssZ'),
     })
-    expect(commit.mock.calls[0]).toEqual([
+    expect(commit.mock.calls[1]).toEqual([
```

Comment:
I find this test very complex. Can  you document it?

Reply:
CHanged it. I did two requests for different parameters, but the first request was checked in previous test, second request needed to be checked in this test.. But I shouldn't have send that irst request again in this test. Fixed that